CDS/js/media-atom-lib.js

```diff
@@ -58,7 +58,9 @@ function fetchMetadata(connection) {
             .then(response => {
                 const title = response.data.title;
                 const description = response.data.description;
-                return Promise.all([datastore.set(connection, 'meta', 'atom_title', title), datastore.set(connection, 'meta', 'atom_description', description)])
+                const categoryId = response.data.categoryId;
```

Comment:
just confirming this is the [youtube category](https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/atoms/media.thrift#L40) rather than the [guardian's category](https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/atoms/media.thrift#L51)?